src/target/source/intrin_rule_webgpu.cc

```diff
@@ -113,6 +113,8 @@ TVM_REGISTER_OP("tir.trunc")
 // extra dispatch
 TVM_REGISTER_OP("tir.erf").set_attr<FLowerIntrinsic>("webgpu.FLowerIntrinsic", DispatchFastErf);
 
+TVM_REGISTER_OP("tir.dot4I8Packed").set_attr<FLowerIntrinsic>("webgpu.FLowerIntrinsic", DispatchPureExtern<Direct>);
```

Comment:
sorry i was not being clear, for tir, it is better to have a common name dp4a

Reply:
Oh I find there is no `tir.dp4a` in TVM right now, and I see in TVM `dp4a` is all called through `call_pure_extern()`: [vulkan](https://github.com/apache/tvm/blob/main/src/target/spirv/codegen_spirv.cc#L382) [cuda](https://github.com/apache/tvm/blob/main/python/tvm/topi/cuda/tensor_intrin.py#L94)

Do you mean we add `tir.dp4a` in TVM or still support `dp4a` as a pure external call like what `dp4a` is supported in `codegen_spirv.cc`?

Reply:
we can add `tir.dp4a` intrinsic, and still support `dp4a` extern

Reply:
@Jiawei-Shao do u mind followup

Reply:
Oh sorry I was busy on some other urgent stuffs these days. I will go back to work on this next week. I will follow the steps to add `tir.dp4a` first.

Reply:
Hi @tqchen,

Sorry for my late response. I've updated this PR. PTAL, thanks!